server/data/data-api/src/test/java/org/apache/james/rrt/lib/CanSendFromContract.java

```diff
@@ -76,6 +75,17 @@ default void userCanSendFromShouldBeTrueWhenSenderIsAnAliasOfAnAliasOfTheUser()
         assertThat(canSendFrom().userCanSendFrom(USER, userAliasBis)).isTrue();
     }
 
+    @Test
+    default void userCannotSendFromShouldBeTrueWhenSenderIsAnAliasOfAnAliasOfAnAliasOfTheUser() throws Exception {
+        Username userAliasBis = Username.of("aliasbis@" + DOMAIN.asString());
+        Username userAliasTer = Username.of("aliaster@" + DOMAIN.asString());
+        addAliasMapping(userAliasTer, userAliasBis);
```

Comment:
I know you are just adding a test (and you are not responsible for the way we do those tests) but could we come with nicer syntax?
What about:
```
redirect(a).to(b);
redirect(b).to(c);
```